server.ts

```diff
@@ -0,0 +1,36 @@
+import * as express from 'express';
+import { TodoApp } from './src/todo';
+
+const todo = new TodoApp();
+
+const serv = express();
+serv.use(express.json());
+
+const port = 3000;
+
+serv.get('/', function(req, res, next) {
```

Comment:
Don't need to fix this but two things come to mind here:

1. Usually if you don't need variables you don't include them.  You use `res` here, and since its the second variable you must define the first (`req`).  But since you don't need or use `next` you don't need to define it here.  More of a scripting thing but good to know.
2. Look up arrow function syntax.  It is what JS and TS are moving toward for anonymous functions.